gui/boosterView.py

```diff
@@ -27,7 +27,8 @@
 
 
 class BoosterViewDrop(wx.PyDropTarget):
-    def __init__(self, dropFn):
+    def __init__(self, dropFn, *args, **kwargs):
+        super(BoosterViewDrop, self).__init__(*args, **kwargs)
```

Comment:
As I understand it, and admittedly this is an area in Python where I'm still weak, when have classes that inherit the `super` call simply makes it explicit.  Yes it's inheriting it, no it's not explicitly doing so, and `explicit is better than implicit`.

The way we're using `super` is the standard basic call, you don't need to  pass `self` because  that's handled for you with the `super(BoosterViewDrop, self)`.

The `__init__` change for the line above the `super` is just to make sure that we match what the parent class is.

If I'm misunderstanding what you're getting at let me know.